src/display/api.js

```diff
@@ -229,10 +235,23 @@ function getDocument(src, pdfDataRangeTransport,
   }
 
   params.rangeChunkSize = params.rangeChunkSize || DEFAULT_RANGE_CHUNK_SIZE;
-  params.disableNativeImageDecoder = params.disableNativeImageDecoder === true;
   params.ignoreErrors = params.stopAtErrors !== true;
   var CMapReaderFactory = params.CMapReaderFactory || DOMCMapReaderFactory;
 
+  if ('disableNativeImageDecoder' in params) {
+    deprecated('parameter disableNativeImageDecoder, ' +
+      'use nativeImageDecoderSupport instead');
+  }
+  params.nativeImageDecoderSupport = params.nativeImageDecoderSupport ||
+    (params.disableNativeImageDecoder === true ? 'none' : 'decode');
+  if (params.nativeImageDecoderSupport !== 'decode' &&
+    params.nativeImageDecoderSupport !== 'none' &&
+    params.nativeImageDecoderSupport !== 'display') {
+    warn('Invalid parameter nativeImageDecoderSupport: ' +
+      'need either `decode`, `none` or `display`');
+    params.nativeImageDecoderSupport = 'decode';
```

Comment:
With comments above taken into account, this line should be:
`params.nativeImageDecoderSupport = NativeImageDecoding.DECODE;`